### DISCUSSION TITLE
How to Find & Filter Issues by Assigner

### DISCUSSION THREAD

**Discussion by iamjonny** [community/community#51759] [Projects and Issues] [answered]
### Select Topic Area

Question

### Body

I searched the discussions and peaked at issue info on rest api docs and nothing came up, although the Web UI shows when I've self assigned, or the assigner when I'm assigned an issue (and get a notification).

I haven't looked up in `gh cli` yet but assume same situation there.

Is it possible to get at the **assigner** field info ?
`is:issue assignee:@me -assigner:username`
Really appreciate any related information. Let me know if I should raise as a Feature Request (and where!)
Apologies in wrong repo, & Thanks In Advance!

**Update** Had a quick search of _the_ org, and found at least one related
https://github.com/search?q=org%3Agithub+assigner&type=issues

and one related reference in the code...
https://github.com/github/docs/blob/1eaae7de50105ba680fc0c8de2ecef9095081fbf/data/reusables/issue-events/assignee-properties.md

> This field is available in the REST API for issue events but not the REST API for timeline events.

**Comment by queenofcorgis** [comment, marked as answer]
Hi there!

This is not a currently available functionality at the moment. However, feel free to use this post to turn it into a feature request. I'll add the appropriate label and if you want to edit your post or leave any comments sharing the use case or any other details to help our team best understand this request that would be mighty helpful. 

Cheers!
